packages/@ama-mfe/messages/package.json

```diff
@@ -41,7 +41,7 @@
     }
   },
   "devDependencies": {
-    "@amadeus-it-group/microfrontends": "^0.0.2",
+    "@amadeus-it-group/microfrontends": "~0.0.3",
```

Comment:
Maybe better to stick with exact version `0.0.3` just in case, not even `~0.0.3`, as there might be breaking changes before `1.0.0`?